devito/symbolics/extended_dtypes.py

```diff
@@ -0,0 +1,123 @@
+import numpy as np
```

Comment:
not sure honestly about this file since after all it's still all about symbolic objects

Reply:
Was annoying me to find it in extended_sympy with all the rest :D can revert